lib/iris/aux_factory.py

```diff
@@ -307,31 +308,6 @@ def _remap_with_bounds(self, dependency_dims, derived_dims):
             nd_values_by_key[key] = nd_values
         return nd_values_by_key
 
-    def _shape(self, nd_values_by_key):
-        nd_values = sorted(nd_values_by_key.values(),
-                           key=lambda value: value.ndim)
-        shape = list(nd_values.pop().shape)
-        for array in nd_values:
-            for i, size in enumerate(array.shape):
-                if size > 1:
-                    # NB. If there's an inconsistency it can only come
-                    # from a mismatch in the number of bounds (the Cube
-                    # ensures the other dimensions must match).
-                    # But we can't afford to raise an error now - it'd
-                    # break Cube.derived_coords. Instead, we let the
-                    # error happen when the derived coordinate's bounds
-                    # are accessed.
-                    shape[i] = size
-        return shape
-
-    def _dtype(self, arrays_by_key, **other_args):
```

Comment:
@pp-mo :scream: Oh my gosh ... this isn't used anywhere! Who knows when it was relevant and useful ... good spot!